Expander.xaml

```diff
@@ -0,0 +1,381 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
+    xmlns:local="using:Microsoft.UI.Xaml.Controls">
+
+    <Style TargetType="local:Expander" BasedOn="{StaticResource DefaultExpanderStyle}"/>
+
+    <Style x:Key="DefaultExpanderStyle" TargetType="local:Expander">
+        <!-- We won't make this control tab stoppable, since the tab focus should go
+            to the toggle button. For this logic, several things were made:
+            1. Expander::OnApplyTemplate - We set the toggle button's events source to the expander's. This will announce expander properties
+               (expand/collapse) changes to narrator. Without this, narrator would announce the toggle button's "on/off" logic
+               instead of the "expand/collapse" logic.
+            2. ExpanderAutomationPeer::GetChildrenCore - Because of 1., we need to override GetChildrenCore to NOT include the toggle button.
+               If we don't do this, we create a weird dependency cycle between the toggle button and the expander.
```

Comment:
There seem to be two Expander.xaml files. This content should be in the one under dev\Expander folder

Reply:
Yes I fixed this